### PR TITLE
dash/KPI-default-value

### PR DESCRIPTION
**Pull request by pawellysy**
Changed which value is prioritised to display.

### CODE REVIEW

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
Test is missing

**Review by Vazonik — APPROVED** [review]
LGTM

File: test/cypress/integration/Dashboards/Components/KPI/KPIComponent.cy.js [hunk 1]
```diff
@@ -4,15 +4,20 @@ describe('KPI Component test', () => {
...
 
     it('KPI Component should have a last value of the point when connector attached.', () => {
-        cy.get('.highcharts-dashboards-component-kpi-value')
-            .should('be.visible')
+        cy.get('#kpi  .highcharts-dashboards-component-kpi-value').as('kpiConnector');
+        cy.get('#kpi-2  .highcharts-dashboards-component-kpi-value').as('kpi-val');
+            cy.get('@kpiConnector').should('be.visible')
             .should('have.text', 1200);
+
+            cy.get('@kpi-val').should('be.visible')
```
Comment (sebastianbochan) [inline test/cypress/integration/Dashboards/Components/KPI/KPIComponent.cy.js:12]: What is the purpose of aliases when it's run one time? Especially in other tests, you do not declare them in similar scenarios.

File: test/cypress/integration/Dashboards/Components/KPI/KPIComponent.cy.js [hunk 2]
```diff
@@ -4,15 +4,20 @@ describe('KPI Component test', () => {
     });
 
     it('KPI Component should have a last value of the point when connector attached.', () => {
-        cy.get('.highcharts-dashboards-component-kpi-value')
-            .should('be.visible')
+        cy.get('#kpi  .highcharts-dashboards-component-kpi-value').as('kpiConnector');
```
Comment (sebastianbochan) [inline test/cypress/integration/Dashboards/Components/KPI/KPIComponent.cy.js:7]: Double spaces.

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Pull request merged**